server/sso/settings.go

```diff
@@ -54,7 +54,9 @@ type Settings struct {
 	// to the IDP
 	AssertionConsumerServiceURL string
 	SessionStore                SessionStore
-	OriginalURL                 string
+	// SessionTTL determines how long (in seconds) a user has to complete SSO before the session is purged from Fleet
+	SessionTTL  uint
```

Comment:
Nit: I think it would be more self-documenting/less error-prone if it was of type `time.Duration` (and convert to integer only where it gets used). I think this PR is somewhat urgent though, so feel free to ignore or address in a follow-up.

Reply:
Putting the conversion here makes the `CreateAuthorizationRequest` call slightly cleaner internally (at the expense of having to do the conversion twice to populate the settings object). If this gets touched again, can tweak that interface, but yeah, this was intended as a high-priority fix while attempting to get the external interface right so we don't have to change/break that later.